src/testers.jl

```diff
@@ -71,6 +71,10 @@ at input point `x` to confirm that there are correct `frule` and `rrule`s provid
 
 `fkwargs` are passed to `f` as keyword arguments.
 All keyword arguments except for `fdm` and `fkwargs` are passed to `isapprox`.
+
+!!! note
```

Comment:
I wonder whether a better option would be to verify that a rule actually _is_ (approximately) holomorphic at the point at which its evaluated using `FiniteDifferences`? That would be a pretty helpful check tbh.

Reply:
To clarify, something like this? Assume function is holomorphic _and_ if passed input is complex also push forward `im` and if received output is complex also pull back `im`, checking that function is holomorphic.

Reply:
Sounds about right to me. Literally check that the jacobian.